### DISCUSSION TITLE
WHMCS Module?

### DISCUSSION THREAD

**Discussion by theozsnowman** [apache/cloudstack#9069] [Q&A] [answered]
Im trying to track down a working module for CloudStack that we can use in our WHMCS installation, mainly for the deployment and management of single instance VM's, but a PAYG type setup could also be good.

Only thing is im having a hard time trying to find any? I found one company but they have done no updates to the module since 2021 and i suspect it doesnt work with the latest releases of either WHMCS or CloudStack

surely there must be some users out there who would be interested in releasing/selling a working module?

can anyone make any suggestions or recommendations?

TIA

**Comment by yadvr** [comment]
Could you care to explain what is WHMCS and how does that relate to CloudStack? If it’s some integration or commercial module you’re better to reach the specific vendor.

**Comment by Pearl1594** [comment, marked as answer]
@theozsnowman You're right, it does seem that the CloudStack module for WHMCS hasn't been maintained for long. Some other solutions that you could possibly look at would be CloudStack module for HostBill or StackConsole, if it meets your requirements.
